about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About our company</title>
+  <link rel="stylesheet" href="assets/dist/css/styles.css">
+  <link rel="stylesheet" href="assets/dist/css/place.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container-fluid container-default">
+    <header class="container-fluid container-default">
+      <div class="top">
+        <img src="assets\icons\facebookWhite.svg" class="icon-top-1" alt="facebook image">
+        <img src="assets\icons\twitter-white.svg" class="icon-top-2" alt="twitter image">
+        <ul class="list-group-top">
+          <li class=""><a href="index.html" class="list-item">English</a></li>
+          <li class="list-item"><a href="index.html" class="list-item">Mypage</a></li>
+          <li class="list-item"><a href="index.html" class="list-item">Logout</a></li>
+        </ul>
+      </div>
+      <div class="nav-together container-desk">
+        <img src="assets/icons/x-icon.svg" class="X-icon" alt="closing menu">
+        <a href="index.html"><img src="assets/images/act/Logo.png" class="logo-image" alt="logo image"></a>
+        <nav class="nav-bar" id="nav-bar">
+          <ul class="nav-list">
+            <li><a href="index.html" class="nav-item">Home</a></li>
+            <li><a href="index.html" class="nav-item">Program</a></li>
+            <li><a href="index.html" class="nav-item">Join</a></li>
+            <li><a href="index.html" class="nav-item">Sponsor</a></li>
+            <li><a href="index.html" class="nav-item">News</a></li>
+          </ul>
+        </nav>
+        <button class="nav-top-button"><a href="about.html">Join the fight!</a></button>
+        </div>
+        <img src="assets/icons/ham-icon.svg" class="ham-icon" alt="menu">
+    </header>
+    <main class="container-fluid container-default aboutpage">
+      <section class="fade-image">
+        <span class="img"></span>
+    </section>
+    <section class="rest-aboutpage">
+      <section class="about-text-section">
+        <h2 class="about-text-2">About Us</h2>
```

Comment:
- [ ] you can use the title you used in the home page and please ensure it is centralized. 
- [ ] please use the normal color,(\#EC5242) as the about page heading has no design
yours 
![Screenshot (1013)_LI](https://user-images.githubusercontent.com/65298048/234985810-e86886e0-2c77-4035-b7a5-e61ad319dfd2.jpg)
original
![Screenshot (1014)](https://user-images.githubusercontent.com/65298048/234985948-8b5804f9-d167-4267-9646-2e3e7a69d57e.png)